src/main/steps/application/task-list/content.ts

```diff
@@ -9,17 +9,10 @@ import {
   getAdoptionCertificateDetailsStatus,
   getApplicationStatus,
   getApplyingWithStatus,
-  getBirthFatherDetailsStatus,
-  getBirthMotherDetailsStatus,
-  getChildrenBirthCertificateStatus,
-  getChildrenPlacementOrderStatus,
   getContactDetailsStatus,
```

Comment:
Remove this file chane, sync with R2master

Reply:
these changes are from me syncing with r2master